prism/src/main/java/org/apache/falcon/service/BacklogMetricEmitterService.java

```diff
@@ -206,6 +216,7 @@ public synchronized void onSuccess(WorkflowExecutionContext context) throws Falc
                 metrics.remove(new MetricInfo(DATE_FORMAT.get().format(date), context.getClusterName()));
                 if (metrics.isEmpty()) {
                     entityBacklogs.remove(entity);
+                    publishBacklog((Process) entity, context.getClusterName(), 0L);
```

Comment:
Making note of the discussion we had offline at the BacklogCalculation we will not have information available which entity got deleted from entityBacklogs so we need to remove it here.